internal/cmd/login.go

```diff
@@ -537,7 +537,7 @@ func promptExistingServers(servers []ClientHostConfig) (string, error) {
 	}
 
 	const defaultOption string = "Connect to a new server"
```

Comment:
don't use const strings, that's not very go-like.
```suggestion
	defaultOption := "Connect to a new server"
```